.bazelrc

```diff
@@ -1,5 +1,6 @@
 startup --expand_configs_in_place
 
+common --@rules_scala_annex//rules/scala:scala-toolchain=zinc_3
```

Comment:
Is this something all users of Annex will need to do? If so, is it in the README for that ruleset?

Reply:
Nope, this is only necessary if you want to override the default Scala toolchain. It's explained here:
https://github.com/lucidsoftware/rules_scala/blob/74960c12dfc2487cd257c2a49d5d4e1c6cfb1d44/docs/newdocs/scala_versions.md

Let me know if I can make it clearer in any way.